dependencyManagement/build.gradle.kts

```diff
@@ -33,6 +34,7 @@ val CORE_DEPENDENCIES = listOf(
   "com.google.errorprone:error_prone_core:${errorProneVersion}",
   "io.github.netmikey.logunit:logunit-jul:2.0.0",
   "io.opentelemetry.proto:opentelemetry-proto:1.0.0-alpha",
+  "io.opentelemetry.semconv:opentelemetry-semconv-incubating:${contribIncubatingVersion}",
```

Comment:
perhaps we should also list `opentelemetry-semconv` here like it is done in the instrumentation repo https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/373224c4c09918de7eaa2aa24e456f94f3975c2f/dependencyManagement/build.gradle.kts#L52

Reply:
That's a good idea, relying on the indirect dependency on the incubating artifact can be confusing.

Reply:
fixed in 18869374843cfbdd60e4e08988a4ae0d87d5cd56